src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/DF_BaseRecalibratorIntegrationTest.java

```diff
@@ -33,7 +33,7 @@ private BQSRTest(String reference, String bam, String knownSites, String args, S
         public String getCommandLine() {
             return  " -R " + reference +
                     " -I " + bam +
-                    " --secretsFile ../client-secrets.json" +
+                    " --secret ../client-secrets.json" +
```

Comment:
In Hellbender's parent directory. We may be able to do without it temporarily (since this code isn't calling the Genomics APIs yet).

This isn't optimal for continuous testing, of course. We might choose to just rely on the API key for a while (works for GCS).